cognite/client/_api/functions.py

```diff
@@ -1041,7 +1024,7 @@ def create(
                 ... )
 
         """
-        _get_function_identifier(function_id, function_external_id)
+        _get_function_identifier(function_id)
```

Comment:
This should no longer use that function, as the error message that can be raised would be misleading:
`"Exactly one of function_id and function_external_id must be specified"`